service/collector/namespace_inconsistency.go

```diff
@@ -0,0 +1,103 @@
+package collector
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/giantswarm/apiextensions/pkg/clientset/versioned"
+	"github.com/giantswarm/helmclient"
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+	"github.com/prometheus/client_golang/prometheus"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/giantswarm/chart-operator/service/controller/chart/key"
+)
+
+var (
+	namespaceInconsistency = prometheus.NewDesc(
+		prometheus.BuildFQName(namespace, "", "namespace_inconsistency"),
+		"namespace is consistent with chart CR spec.",
```

Comment:
```suggestion
		"namespace is inconsistent with chart CR spec.",
```
Should this match the metric name?


Naming is hard here but I'm nor sure about inconsistency. Could we use mismatch? As it is also shorter.

But I'm also not sure that is better. So go with what you think is best.

Reply:
`mismatch` is a lot better I think, I will change the name to that. Thanks for the input!
`inconsistency` felt like too much scientific term 😆 .